src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_params.py

```diff
@@ -104,6 +104,14 @@ def get_hostname_completion_list(prefix, action, parsed_args, **kwargs): # pylin
 register_cli_argument('appservice web config update', 'java_container_version', help="The version of the java container, e.g., '8.0.23' for Tomcat")
 register_cli_argument('appservice web config update', 'app_command_line', options_list=('--startup-file',), help="The startup file for linux hosted web apps, e.g. 'process.json' for Node.js web")
 
+register_cli_argument('appservice web config ssl bind', 'certificate_thumbprint', help='The ssl cert thumbprint')
+register_cli_argument('appservice web config ssl bind', 'ssl_type', help='The ssl cert type, e.g., SNI or IP', **enum_choice_list(['SNI', 'IP']))
```

Comment:
If I import the enum...

Found in \env\Lib\site-packages\azure\mgmt\web\models\web_site_management_client_enums.py
```
class SslState(Enum):
    disabled = "Disabled"
    sni_enabled = "SniEnabled"
    ip_based_enabled = "IpBasedEnabled"
```
I would need a mapping from two values out of the three values to two strings otherwise the third enum would be undesirably exposed. I prefer to keep a static list for now.